File: revsision/V2/MaxOnes.py
def max_ones(list):
    one_count=0;
    max_one_count=float("-inf")
    for elem in list:
        if(elem==1):
            one_count+=1
            max_one_count=max(one_count,max_one_count)
        else:
            one_count=0
    return max_one_count

print(max_ones([1,1,0,1,1,1,]))

File: revsision/V2/MaxOnes.java
public class MaxOnes {
    public static int maxOnes(int[] arr) {
        int maxCount = Integer.MIN_VALUE;
        int oneCount = 0;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == 1) {
                oneCount++;
                maxCount = Math.max(oneCount, maxCount);
            } else {
                oneCount = 0;
            }
        }
        return maxCount;
    }

    public static void main(String[] args) {
        System.out.println(maxOnes(new int[] { 1, 1, 0, 1, 1, 1 }));
    }
}
